src/lib/gssapi/krb5/iakerb.c

```diff
@@ -1041,11 +1046,33 @@ iakerb_gss_init_sec_context(OM_uint32 *minor_status,
         if (ctx->gssc == GSS_C_NO_CONTEXT)
             input_token = GSS_C_NO_BUFFER;
 
+        atarget_name = (gss_name_t) target_name;
```

Comment:
You already set this in line 974 and 999.

Reply:
fixed